elasticdl/python/worker/worker_pytorch.py

```diff
@@ -0,0 +1,421 @@
+# Copyright 2020 The ElasticDL Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import time
+import traceback
+
+import numpy as np
+import tensorflow as tf
+from tensorflow.keras import backend as K
+
+from elasticdl.proto import elasticdl_pb2
+from elasticdl.python.collective_ops.communicator import CollectiveCommunicator
+from elasticdl.python.common.constants import (
+    CollectiveCommunicatorStatus,
+    Initializer,
+    JobType,
+    MetricsDictKey,
+    Mode,
+)
+from elasticdl.python.common.dtypes import dtype_numpy_to_tensor
+from elasticdl.python.common.log_utils import get_logger
+from elasticdl.python.common.model_handler import ModelHandler
+from elasticdl.python.common.model_utils import (
+    find_layer,
+    get_dict_from_params_str,
+    get_model_spec,
+    get_non_embedding_trainable_vars,
+    set_callback_parameters,
+)
+from elasticdl.python.common.tensor_utils import EmbeddingTableInfo, Tensor
+from elasticdl.python.common.timing_utils import Timing
+from elasticdl.python.elasticdl.callbacks import SavedModelExporter
+from elasticdl.python.elasticdl.feature_column import feature_column
+from elasticdl.python.elasticdl.layers.embedding import Embedding
```

Comment:
Do you use `elasticdl.python.elasticdl.layer`, `elasticdl.python.elasticdl.callbacks`, `elasticdl.python.elasticdl.feature_column` in the pytorch training loop?

Reply:
Actually, these packages are not used. Based training loop doesn't need it.

Reply:
I will remove them later.